idoc/__init__.py

```diff
@@ -1,3 +1,3 @@
 from . import ilqr, tilqr, lqr, qp, sqp, typs, utils
 
-__all__ = ["ilqr", "tilqr", "lqr", "qp", "sqp", "typs", "utils"]
+__all__ = ["ilqr", "tilqr", "lqr", "qp", "sqp", "typs", "batch_lqr", "batch_ilqr", "utils"]
```

Comment:
in terms of file names, shall we do `blqr`, `bilqr` instead of `batch_lqr`, `batch_ilqr`?